zold.gemspec

```diff
@@ -71,4 +71,5 @@ and suggests a different architecture for digital wallet maintenance.'
   s.add_development_dependency 'rdoc', '4.2.0'
   s.add_development_dependency 'rspec-rails', '3.1.0'
   s.add_development_dependency 'webmock', '3.4.2'
+  s.add_development_dependency 'byebug'
```

Comment:
@mgrassotti why don't you name the version of it, like it's done for all other dependencies? Any particular reason?